packages/ethereum/contracts/HoprChannels.sol

```diff
@@ -37,6 +37,7 @@ contract HoprChannels is IERC777Recipient, ERC1820Implementer {
          * stateCounter mod 10 == 2: open
          * stateCounter mod 10 == 3: pending
          */
+        bool closureByPartyA; // channel closure was initiated by party A
     }
```

Comment:
Here we should decrease the size of a previous entry in the struct to keep the total size of the struct to <= 64 bytes (2 words). I suggest to decrease the `closureTime` to `uint32` which is going to be fine until Sunday, February 7, 2106 6:28:15 AM.